pwa/src/services/dateTime.tsx

```diff
@@ -19,3 +19,19 @@ export const dateTime = (language: TLanguageCodes, dateTime: string) => {
     </div>
   );
 };
+
+export const dateTimeTag = (language: TLanguageCodes, dateTime: string) => {
+  if (!dateTime) return;
+
+  let _dateTime = Date.parse(dateTime);
+  const newDate = new Date(_dateTime);
+
+  const _language = languageOptions.find((_language) => _language.label === _.upperCase(language));
+
+  const localDate = newDate.toLocaleDateString(_language?.value);
+  const localTime = newDate.toLocaleTimeString(_language?.value);
+
+  const returnValue = `${localDate} | ${localTime}`;
+
+  return returnValue;
```

Comment:
Maybe we could simply return `${localDate} ....` instead of assigning it and then returning the const.